spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaAutoConfiguration.java

```diff
@@ -60,6 +63,9 @@
     private static final Port DEFAULT_PORT = new Port().setPort(8080)
                                                        .setProtocol(SessionProtocol.HTTP);
 
+    @Autowired(required = false)
+    private DocServiceBuilderSupplier docServiceBuilderSupplier;
```

Comment:
You could instead move this into a parameter of `armeriaServer()` method for consistency, because other components are injected there.

Reply:
ok